generators/chipyard/src/main/scala/config/AbstractConfig.scala

```diff
@@ -45,6 +45,8 @@ class AbstractConfig extends Config(
   new chipyard.config.WithNoSubsystemDrivenClocks ++             // drive the subsystem diplomatic clocks from ChipTop instead of using implicit clocks
   new chipyard.config.WithInheritBusFrequencyAssignments ++      // Unspecified clocks within a bus will receive the bus frequency if set
   new chipyard.config.WithPeripheryBusFrequencyAsDefault ++      // Unspecified frequencies with match the pbus frequency (which is always set)
+  new chipyard.config.WithMemoryBusFrequency(100.0) ++           // Default 100 MHz mbus
```

Comment:
Does this affect the default config's simulation speed?

Reply:
No. The default was already 100 MHz here, this just makes it more explicit.

Reply:
The explicitness is necessary for the WithSimDRAM HarnessBinder to pick it up